lua/conform/formatters/templ.lua

```diff
@@ -0,0 +1,9 @@
+---@type conform.FileFormatterConfig
+return {
+  meta = {
+    url = "https://templ.guide",
```

Comment:
Could you link to the fmt command directly? https://templ.guide/commands-and-tools/cli/#formatting-templ-files

Reply:
Updated the URL in Readme and lua file.